File: AspNetMvcUnleashed/0672329980_source/Chapter03/VB/MvcApplication1/MvcApplication1/Controllers/ProductController.vb
Public Class ProductController
    Inherits System.Web.Mvc.Controller

    '
    ' GET: /Product/

    Function Index() As ActionResult
        Return View()
    End Function

    '
    ' GET: /Product/Help

    Function Help() As ActionResult
        Return View()
    End Function

    '
    ' GET: /Details/1

    Function Details(ByVal id As Integer) As ActionResult
        Return View()
    End Function


End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter03/CS/MvcApplication1/MvcApplication1/Controllers/ProductController.cs
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class ProductController : Controller
    {
        //
        // GET: /Product/

        public ActionResult Index()
        {
            return View();
        }

        //
        // GET: /Product/Help

        public ActionResult Help()
        {
            return View();
        }

        //
        // GET: /Details/1

        public ActionResult Details(int Id)
        {
            return View();
        }


    }
}
